WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -126,11 +126,29 @@ public function getGroups() {
 				),
 			),
 
+			'unlink' => array(
+				'type'      => 'warning',
+				'message'   => '%s() is discouraged. Use wp_delete_file() for remote URLs instead.',
+				'since'     => 'x.x.x',
+				'functions' => array(
+					'unlink',
+				),
+			),
+
 			'file_system_read' => array(
 				'type'      => 'warning',
 				'message'   => 'File operations should use WP_Filesystem methods instead of direct PHP filesystem calls. Found: %s()',
 				'since'     => '2.5.0',
 				'functions' => array(
+					'chgrp',
+					'chmod',
+					'chown',
+					'is_writable',
+					'is_writeable',
+					'mkdir',
+					'rename',
```

Comment:
Based on https://github.com/WordPress/WordPress-Coding-Standards/pull/1265#issuecomment-353495333, the WP_FileSystem counterpart for `rename` would be `move`, so should this be in a a separate group so the replacement can be mentioned ?

Reply:
Added in separate group